addons/product/migrations/13.0.1.2/post-migration.py

```diff
@@ -36,8 +36,11 @@ def fill_product_variant_combination_table(env):
         (product_product_id, product_template_attribute_value_id)
         SELECT pavppr.product_product_id, ptav.id
         FROM product_attribute_value_product_product_rel pavppr
-        JOIN product_template_attribute_value ptav ON
-            ptav.product_attribute_value_id = pavppr.product_attribute_value_id
+        JOIN product_product pp
+            ON pp.id = pavppr.product_product_id
```

Comment:
What happen if there are no product variants populated yet?

Reply:
OK, I see there's no problem on that.

Reply:
we have linked product template attribute value which not related for this product template

![image](https://user-images.githubusercontent.com/33932458/85418476-7e214000-b579-11ea-8491-48a7a7453a4e.png)
![image](https://user-images.githubusercontent.com/33932458/85418984-2fc07100-b57a-11ea-915d-80f42aa9ce82.png)